frontend/src/container/GeneralSettings/StatusMessage.tsx

```diff
@@ -43,9 +43,16 @@ function StatusMessage({
 				width: '100%',
 			}}
 		>
-			<Row style={{ gap: '1rem', alignItems: 'center', justifyContent: 'center' }}>
+			<div
+				style={{
```

Comment:
Can we move this styling to another files.

Reply:
It's previous code, changed from Row -> div